src/main/resources/assets/paths/user.yaml

```diff
@@ -52,6 +52,8 @@ put:
             suggestedSigningOfficial:
               description: The suggested signing official for the user
               type: string
+            daaAcceptance:
+              type: string
```

Comment:
Nit: Add a description to the docs as well. For context, `daaAcceptance` represents whether or not a user has accepted the Broad and NIH's Data Access Agreement.